text_formatter.go

```diff
@@ -331,9 +337,14 @@ func (f *TextFormatter) appendValue(b *bytes.Buffer, value interface{}) {
 		stringVal = fmt.Sprint(value)
 	}
 
-	if !f.needsQuoting(stringVal) {
+	// write value without quoting directly when ExcludeKey is true
+	if f.ExcludeKey {
```

Comment:
If configure ExcludeKey  as true, just write value without quoting directly, otherwise, the logs will be as below
```
"0001-01-01T00:00:00Z" warning "message"
```